<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
<<<<<<< HEAD
    <width>1293</width>
    <height>942</height>
=======
    <width>920</width>
    <height>744</height>
>>>>>>> fa77c25b74643853fa49c8d8614903b4253493c7
   </rect>
  </property>
  <property name="windowTitle">
   <string>UK5com</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="UK5Q_progressBar">
    <property name="geometry">
     <rect>
      <x>0</x>
<<<<<<< HEAD
      <y>890</y>
      <width>1291</width>
=======
      <y>700</y>
      <width>921</width>
>>>>>>> fa77c25b74643853fa49c8d8614903b4253493c7
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
<<<<<<< HEAD
   <widget class="QWidget" name="layoutWidget">
=======
   <widget class="QWidget" name="">
>>>>>>> fa77c25b74643853fa49c8d8614903b4253493c7
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
<<<<<<< HEAD
      <width>1301</width>
=======
      <width>921</width>
>>>>>>> fa77c25b74643853fa49c8d8614903b4253493c7
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_LEFT">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="UK5Q_label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Расчет кратного разбавления</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_RIGHT">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
<<<<<<< HEAD
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>840</y>
      <width>641</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_BUTTONS">
     <item>
      <spacer name="horizontalSpacer_LEFT2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="UK5Q_Eval">
       <property name="text">
        <string>Вычислить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="UK5Q_Exit">
       <property name="text">
        <string>Выйти</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_RIGHT2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>820</y>
      <width>641</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_LIST">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_XLS">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_SRF">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_RES">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
=======
>>>>>>> fa77c25b74643853fa49c8d8614903b4253493c7
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
<<<<<<< HEAD
      <width>1291</width>
      <height>781</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
=======
      <width>921</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_MAIN">
>>>>>>> fa77c25b74643853fa49c8d8614903b4253493c7
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_LEFT">
       <item>
        <widget class="QGroupBox" name="groupBox_IN">
         <property name="title">
          <string/>
         </property>
         <widget class="QScrollArea" name="scrollArea_IN">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
<<<<<<< HEAD
            <width>641</width>
=======
            <width>451</width>
>>>>>>> fa77c25b74643853fa49c8d8614903b4253493c7
            <height>581</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="UK5Q_scrollAreaWidgetContents_IN">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
<<<<<<< HEAD
             <width>639</width>
=======
             <width>449</width>
>>>>>>> fa77c25b74643853fa49c8d8614903b4253493c7
             <height>579</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_IN_OUT">
         <property name="title">
          <string/>
         </property>
         <widget class="QScrollArea" name="scrollArea_IN_OUT">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
<<<<<<< HEAD
            <width>641</width>
=======
            <width>451</width>
>>>>>>> fa77c25b74643853fa49c8d8614903b4253493c7
            <height>581</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="UK5Q_scrollAreaWidgetContents_IN_OUT">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
<<<<<<< HEAD
             <width>639</width>
=======
             <width>449</width>
>>>>>>> fa77c25b74643853fa49c8d8614903b4253493c7
             <height>579</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_RIGHT">
       <item>
        <widget class="QGroupBox" name="groupBox_OUT">
         <property name="title">
          <string/>
         </property>
         <widget class="QScrollArea" name="scrollArea_OUT">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
<<<<<<< HEAD
            <width>641</width>
=======
            <width>451</width>
>>>>>>> fa77c25b74643853fa49c8d8614903b4253493c7
            <height>581</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="UK5Q_scrollAreaWidgetContents_OUT">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
<<<<<<< HEAD
             <width>639</width>
=======
             <width>449</width>
>>>>>>> fa77c25b74643853fa49c8d8614903b4253493c7
             <height>579</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_CHART">
         <property name="title">
          <string/>
         </property>
         <widget class="QChartView" name="UK5Q_Chart">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
<<<<<<< HEAD
            <width>641</width>
            <height>211</height>
=======
            <width>451</width>
            <height>151</height>
>>>>>>> fa77c25b74643853fa49c8d8614903b4253493c7
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
<<<<<<< HEAD
=======
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>640</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_LIST">
     <item>
      <widget class="QGroupBox" name="groupBox_XLS">
       <property name="title">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_SRF">
       <property name="title">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_RES">
       <property name="title">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>644</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_BUTTONS">
     <item>
      <spacer name="horizontalSpacer_LEFT2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="UK5Q_Eval">
       <property name="text">
        <string>Вычислить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="UK5Q_Exit">
       <property name="text">
        <string>Выйти</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_RIGHT2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
>>>>>>> fa77c25b74643853fa49c8d8614903b4253493c7
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
